### ISSUE TITLE
[Problem] Unable to install Cerberus 3.6

### ISSUE THREAD

**Issue by joe675** [cerberustesting/cerberus-source#1782]
Hello everyone,

I would like to install Cerberus on Windows 10 but I have this issue :

> cerberus-as-glassfish_1  | * Starting Cerberus Glassfish setup...
cerberus-as-glassfish_1  | Waiting for domain1 to start .........
cerberus-as-glassfish_1  | Successfully started the domain : domain1
cerberus-as-glassfish_1  | domain  Location: /glassfish5/glassfish/domains/domain1
cerberus-as-glassfish_1  | Log File: /glassfish5/glassfish/domains/domain1/logs/server.log
cerberus-as-glassfish_1  | Admin Port: 4848
cerberus-as-glassfish_1  | Command start-domain executed successfully.
cerberus-as-glassfish_1  | **Authorization has been refused for credentials [user: admin] given in this request.**
cerberus-as-glassfish_1  | **(Usually, this means invalid user name and/or password)**
cerberus-as-glassfish_1  | **Command change-admin-password failed.**

I use the "docker-compose up" with the default.yml file

Could someone solve this problem ?

**Comment by yun59000** [comment]
Hi,
i had the same issue and after some tries i've found a solution:
you have to modify your default.yml  
check the image line for "cerberus-as-glassfish:"  you need to have at least "cerberustesting/cerberus-as-glassfish:3.7.1"

**Comment by nicodeur** [comment]
Hi,

Effectively, There was a problem with password admin, It fix by  cerberus-as-glassfish `3.7.1` 

Solution to get arround problem on the lower version to `3.7.1` is to add environment variable on `cerberus-as-glassfish`  :

```
 cerberus-as-glassfish:
  image: cerberustesting/cerberus-as-glassfish:latest
  .....
  environment:
   - GLASSFISH_DOMAIN=cerberus
  .....
```
 
For information, we had  a problème on open a volume on glassfish domain. It fix since `3.7.1`too. Now, you can add this 2 volumes : 
```
 volumes:
  - ./localdata/cerberus-domains:/glassfish5/glassfish/domains/cerberus # allow to persist database pool connection / custom glassfish settings
  - ./localdata/lib:/tmp/lib/ # allow to add jdbc library to connect database to cerberus
```

**Comment by joe675** [comment]
Thanks for your help, it works now !
